src/components/CreateAnalysis/ConfirmAnalysis.tsx

```diff
@@ -6,7 +6,7 @@ import CreateAnalysisService from "../../services/CreateAnalysisService";
 import ImageSelectionBox from "./ImageSelectionBox";
 
 interface ConfirmAnalysisProps{
-  submit: () => void
+  submit: (XrayModel: string | void, CTModel: string | void) => (() => void)
```

Comment:
Did you mean `undefined` instead of `void`?

If yes, you could change the line as follows
```suggestion
  submit: (XrayModel?: string, CTModel?: string) => (() => void)
```

Reply:
Great suggestion, makes it much cleaner to write/read